### PR TITLE
Add Tampl, Pixelbox Lite, pngLua, Acidity and GEMU

### PR DESCRIPTION
**Pull request by 9551-Dev**
- [x] I have read the [contribution guidelines](./CONTRIBUTING.md).
- [x] I agree to release my contributions under the [CC-BY-SA](./LICENSE.md) license.
- [x] I have read the automated feedback from the spell-checker after submitting this pull request. 

## Summary of changes

Add Tampl and pixelbox_lite and pngLua and Acidity and Gemu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
 - [ComputerCraft Advanced Shell (cash)](https://cash.madefor.cc) -  A Bourne-compatible shell for ComputerCraft. 
 - [Consult](https://consult.madefor.cc) -  A text editor emphasizing ease of use and compatibility with many systems. 
     - [Consult: Recrafted](https://github.com/manaphoenix/CONSULT_RECRAFTED) -  A fork of Consult for Recrafted systems.
-- [gist](https://pastebin.com/zSLPYpqs) -  A GitHub Gist download/upload program which aims to extend the built-in `pastebin` program, but for Gists. 
+- [GEMU](https://github.com/9551-Dev/GEMU) -  CraftOS pc graphics mode emulator based on pixelbox
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- [GEMU](https://github.com/9551-Dev/GEMU) -  CraftOS-PC graphics mode emulator based on Pixelbox.
```

File: README.md [hunk 2]
```diff
@@ -78,6 +79,7 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
 
 ### Library
 
+- [Acidity](https://github.com/9551-Dev/acidity) -  A simple to use [procedual world generation](https://cdn.discordapp.com/attachments/477911902152949771/1078015614712565770/image.png) oriented noise generation library
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- [Acidity](https://github.com/9551-Dev/acidity) -  A simple [procedual world generation](https://cdn.discordapp.com/attachments/477911902152949771/1078015614712565770/image.png)-oriented noise generation library.
```

File: README.md [hunk 3]
```diff
@@ -96,8 +98,11 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
 - [Luz](https://github.com/MCJack123/Luz) - An extremely light compression algorithm for Lua files.
 - [Milo](https://github.com/kepler155c/opus-apps/wiki/Milo-(crafting---storage-system)) -  A crafting and inventory management system. Note that this depends on OpusOS and Plethora.
 - [Pine3D](https://github.com/Xella37/Pine3D) - A 3D rendering engine designed to be fast.
+- [PngLua](https://github.com/9551-Dev/pngLua) -  Png image parsing library designed for computercraft
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- [PngLua](https://github.com/9551-Dev/pngLua) -  PNG parsing library designed for ComputerCraft.
```

File: README.md [hunk 4]
```diff
@@ -96,8 +98,11 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
...
+- [Pixelbox lite](https://github.com/9551-Dev/apis/blob/main/pixelbox_lite.lua) - An insanely fast [drawing character](https://cdn.discordapp.com/attachments/876936991256743947/1123213991813058751/image.png) processing library
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- [Pixelbox Lite](https://github.com/9551-Dev/apis/blob/main/pixelbox_lite.lua) - An insanely fast [drawing character](https://cdn.discordapp.com/attachments/876936991256743947/1123213991813058751/image.png) processing library.
```

File: README.md [hunk 5]
```diff
@@ -96,8 +98,11 @@ A note on abbreviations: `CC` is ComputerCraft, `CC:T` is ComputerCraft: Tweaked
...
 - [RedRun](https://gist.github.com/MCJack123/473475f07b980d57dd2bd818026c97e8) -  A small library for running processes in the background of CraftOS after exiting the program, similar to DOS TSRs.
 - [Tamperer](https://github.com/Fatboychummy-CC/Tamperer) -  A library allowing easy creation of settings menus.
+- [Tampl](https://github.com/9551-Dev/tampl) -  Lua tokenization/parsing library designed for injecting code into existing programs and dynamic code generation
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- [Tampl](https://github.com/9551-Dev/tampl) - A Lua tokenization and parsing library designed for dynamic code generation and injecting code into existing programs.
```

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
Thanks for contributing. Please correct the changes I've pointed out above. Please then rerun the awesome-lint linter per the contributing guidelines and fix any issues with the lines you've added. (Feel free to ignore the spellchecker, I'm in the process of replacing it with a better one that allows linting things like missing punctuation and common misspellings.)

(Lurkers: Feel free to point out any issues with this PR, I'm on mobile so I won't be as thorough as I would be on a laptop.)

**Comment by 9551-Dev** [comment]
man im sorry for leaving so many mistakes lol but i coulnt figure out how to modify a PR with git lmao

**Comment by 9551-Dev** [comment]
honestly not sure what PR spellcheck is complaining about now?

**Comment by tomodachi94** [comment]
@9551-Dev me neither, I've been meaning to replace it with something *much* better. I'm going to merge this in with a squash.

Thank you for your contribution!

(P.S. you can amend a commit by running `git commit --amend`.)

**Pull request merged**